MarvellOS.py

```diff
@@ -6,7 +6,16 @@
 from tkinter import *
 import base64
 
-WINDOW_DIM = "325x400"
+
+#Tkinter Related
+
+import tkinter as tk
+import tkinter.filedialog as tkfd
+from tkinter import *
+from tkinter import ttk 
+
+
+WINDOW_DIM = "400x500"
```

Comment:
I don't see a need to change the default window size. We'll fix the positioning and sizing of elements separately, so you can revert to the default size.